OneCommunityGlobal/HighestGoodNetworkApp
# Ruike_add_additional_condition_to_fix_bug

# Description
<img width="735" alt="Screen Shot 2023-11-11 at 10 34 55 PM" src="https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/114443664/cc81353c-e6e6-4adb-9bd6-88084d9815e9">


## Related PRS (if any):
N/A

## Main changes explained:
Add additional condition to ignore tasks that is not assigned or not active.

## How to test:
1. check into current branch
2. do `npm install` and `...` to run this PR locally
3. log in as admin or owner
4. assign task to a volunteer
5. log out and log in as volunteer
6. make sure the default tab is tasks for volunteer right now
7. log out and log in as admin or owner again
8. set the task to be unassigned
9. log out and log in as volunteer
10. make sure the default tab is Current Week Timelog

## Screenshots or videos of changes:
Before:

https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/114443664/8d0b3fc6-b5c8-423e-8972-47264b675832
